### ISSUE TITLE
does not work

### ISSUE THREAD

**Issue by martijintro** [oskopek/javaanpr#84]
F:\Downloads>java -jar javaanpr.jar
kein Hauptmanifestattribut, in javaanpr.jar

**Comment by martijintro** [comment]
problem is not a problem anymore - so it seems, your software does not work with swiss catr plate numbers

**Comment by martijintro** [comment]
"so it seems, your software does not work with swiss car plate numbers"

Maybe you can confirm that? Thank you very much, then i know what the facts are.

**Comment by oskopek** [comment]
Hi!

Unfortunately, I don't know if it works or doesn't with Swiss license plates. There is no "design issue" from preventing it to work with Swiss license plates, to the best of my knowledge. If it doesn't work in your case, you can try the steps in [this doc](https://github.com/oskopek/javaanpr/blob/master/docs/howto-use-reports.adoc). Of course, you're welcome to submit any improvements you make if you get it to work on your example :)

**Comment by martijintro** [comment]
Hi

Thank you very much for your feedback. The major issue seems to be on 2-line swiss cat plates: The canton (similar to "province") on the 1st line, the number itself on the 2nd line. That kind of number are existing (too) in switzerland - no just the ones with a single line:

![sdfgasergf](https://user-images.githubusercontent.com/42765391/78755951-48a58700-797a-11ea-8ba8-96ce45d3cd23.jpg)

![unnamed (1)](https://user-images.githubusercontent.com/42765391/78755952-493e1d80-797a-11ea-90a8-ad258603fc4b.jpg)

**Comment by oskopek** [comment]
Yeah, as far as I know, 2 line plates don't work by default. Technically, the same method should work, but you would have to do code changes. Sorry for that :/
